src/store/modules/actions/staticData.js

```diff
@@ -135,3 +135,24 @@ export async function lazyGetNFTOwners({ getters, dispatch }, classId) {
   }
   return owners;
 }
+
+export async function fetchLIKEPrice({ commit }) {
+  let price = 0;
+  try {
+    const data = await this.$api.$get(api.getLIKEPrice());
+    price = data.likecoin.usd;
+    commit(TYPES.STATIC_SET_LIKE_PRICE_IN_USD, price);
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.error('CANNOT_GET_LIKE_PRICE');
```

Comment:
Maybe return the previous price or rough price (if no previous price )?
https://github.com/likecoin/liker-land/blob/af8403b5fddbe408b59d9994ddad3833d99aa687/src/constant/index.js#L106